webui/react/src/shared/components/message/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './Message';
+import Message from './Message';
+export default Message;
```

Comment:
nice, you got both named and default re-exported

Reply:
I wonder if there's a shorter syntax for rexporting the default export.

Reply:
I looked into this too when I saw this section, I think this is the most succinct way.